packages/replay/metrics/test-apps/jank/with-sentry.html

```diff
@@ -0,0 +1,56 @@
+<!--
```

Comment:
Could we have three scenarios: Plain, with sentry, with sentry + replay? 
Would be very interesting to know how much of the overhead is sentry itself vs. replay!

Reply:
yep, made the change, see the screenshot and link to the comment in the PR description.